spec/forms/link_case/confirmation_form_spec.rb

```diff
@@ -0,0 +1,125 @@
+require "rails_helper"
+
+RSpec.describe LinkCase::ConfirmationForm, type: :form do
+  subject(:instance) { described_class.new(params) }
+
+  let(:params) do
+    {
+      model: linked_application,
+      link_type_code:,
+    }
+  end
+
+  let(:legal_aid_application) { create(:legal_aid_application) }
+  let(:linked_application) { create(:linked_application, lead_application_id: source_application.id, associated_application_id: legal_aid_application.id) }
+
+  let(:source_application) do
+    create(:legal_aid_application,
+           application_ref: "L-TVH-U0T",
+           provider: legal_aid_application.provider)
+  end
+
+  describe "#save" do
+    subject(:call_save) { instance.save }
+
+    context "with family link chosen" do
+      let(:link_type_code) { "FC_LEAD" }
+
+      it "updates the link_type_code" do
+        expect { call_save }.to change(linked_application, :link_type_code).from(nil).to("FC_LEAD")
+      end
```

Comment:
we should probably have a context or contexts to capture the flows whereby `legal_aid_applicaiton#link_case` is updated too:

```ruby
context "when linking a copied case" do
  ...setup...
  
  it "updates link_case to true" do
    expect { call_save }.to change { legal_aid_applicaiton.link_case }.from(nil).to(true)
  end
end
```